### DISCUSSION TITLE
The `#{!important}` class does not exist. If `#{!important}` is a custom class, make sure it is defined within a `@layer` directive.

### DISCUSSION THREAD

**Discussion by rupesh45** [tailwindlabs/tailwindcss#6994] [Help]
I'm using sass with tailwindcss and I want to use important!

:host ::ng-deep ss-multiselect-dropdown .open>.dropdown-menu {
    @apply flex #{!important};
    
}

This does not work, throws error!

**Comment by Cephyric-gh** [comment]
You've missed some quotes - `#{'!important'}` (the `!important` part needs to be in quotes so it outputs as a string)

**Reply by rupesh45** [reply]
The `#{'!important'}` class does not exist. If `#{'!important'}` is a custom class, make sure it is defined within a `@layer` directive.
  31 | :host ::ng-deep ss-multiselect-dropdown .open>.dropdown-menu {
          @apply flex #{'!important'};
          ^

Throws this error!

**Reply by Cephyric-gh** [reply]
This looks like Angular, and based on the selector pathing you're trying to add this to a component rather than a root CSS file? If that's the case then you have a few options:

- If you're on Angular 12/13 and wish to support writing SASS in your component styles then you need to add `"inlineStyleLanguage": "scss”` to your project settings in your `angular.json`. Once you've added that option your component styles should all be parsed as SASS (which of course allows you to reap all the benefits of SASS), however you'll probably want to check through your entire project going this route as some of your components may now be getting their styles parsed differently to how they did in the past.
- If you don't want to change your project setup, then your component styles are parsed as plain CSS, which means you should be able to just use `@apply flex !important`
